modules/custom/falcon_donation/src/EventSubscriber/FalconDonationOrderCompletedSubscriber.php

```diff
@@ -66,6 +66,11 @@ public function orderCompleteHandler(WorkflowTransitionEvent $event) {
       }
 
       if ($order->get('field_appeal')->isEmpty()) {
+        $this->logger
+          ->info(
```

Comment:
1. It's an error, not info.

2. The text should be `Could not send Thank You email, because the donation order does not have reference to the appeal. Related order ID is @id.`